nixos/modules/services/web-apps/chasemapper.nix

```diff
@@ -0,0 +1,523 @@
+            description = ''
+              Car heading speed gate
+            '';
+            type = lib.types.int;
+          };
+          turn_rate_threshold = lib.mkOption {
+            default = 4.0;
+            description = ''
+              Turn rate threshold
+            '';
+            type = lib.types.float;
+          };
+          bearing_length = lib.mkOption {
+            default = 10;
+            description = ''
+              Bearing length in km
+            '';
+            type = lib.types.int;
+          };
+          bearing_weight = lib.mkOption {
+            default = 1.0;
+            description = ''
+              Weight of the bearing lines, in pixels
+            '';
+            type = lib.types.float;
+          };
+          bearing_color = lib.mkOption {
+            default = "red";
+            description = ''
+              Color of the bearings
+            '';
+            type = lib.types.str;
+          };
+          bearing_custom_color = lib.mkOption {
+            default = "#FF0000";
+            description = ''
+              Custom bearing color, in hexadecimal #RRGGBB
+            '';
+            type = lib.types.str;
+          };
+        };
+
+        units = {
+          unitselection = lib.mkOption {
+            default = "metric";
+            description = ''
+              unitselection allows choice of metric
+            '';
+            type = lib.types.str;
+          };
+          switch_miles_feet = lib.mkOption {
+            default = 400;
+            description = ''
+              This is the threshold for switching from miles to feet, set in metres.
+            '';
+            type = lib.types.int;
+          };
+        };
+
+        history = {
+          reload_last_position = lib.mkOption {
+            default = false;
+            description = ''
+              Enable load of last position from log files
+            '';
+            type = lib.types.bool;
+          };
+        };
+      };
+
+      openPorts = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = "Whether to open firewall ports for chasemapper";
+      };
+
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    systemd.services.chasemapper = {
+      serviceConfig = {
+        Type = "simple";
+        Restart = "always";
+
+        DynamicUser = true;
+        PrivateTmp = true;
+        WorkingDirectory = cfg.package;
+        User = cfg.user;
+        Group = cfg.group;
+        ExecStart = "${exe} --nolog -c ${chaseCfg}";
+      };
+
+      after = [
+        "network.target"
+      ];
+      description = "Chasemapper";
+      wantedBy = [ "multi-user.target" ];
+    };
+
+    networking.firewall.allowedTCPPorts = lib.mkIf cfg.openPorts [
+      cfg.settings.map.flask_port
+    ];
+
+    users.users.${cfg.user} = {
+      group = "${cfg.group}";
+      isSystemUser = true;
+    };
+
+    users.groups.${cfg.group} = { };
```

Comment:
Why this is required? You're using `DynamicUser`, why would we need to have a specific user ?